hadoop-mapreduce/src/main/java/org/apache/accumulo/hadoopImpl/mapreduce/AccumuloOutputFormatImpl.java

```diff
@@ -88,7 +88,7 @@ public static void setClientInfo(Job job, ClientInfo info) {
    *
    * @since 2.0.0
    */
-  protected static ClientInfo getClientInfo(JobContext context) {
+  public static ClientInfo getClientInfo(JobContext context) {
```

Comment:
Should just drop this method and call the OutputConfigurator.

Reply:
I am not sure why this should be done.  Do you want to create a separate issue?

Reply:
Yeah I will open an issue.  There is just a lot of redundant code now in the new map reduce impl that we needed before because of so many methods being in the public API.